ha/core/cluster/cluster_manager.py

```diff
@@ -294,73 +275,61 @@ def status(self):
     def shutdown(self):
         raise HAUnimplemented("This feature is not supported...")
 
-class CortxClusterManager:
+class CortxClusterManager(ClusterManager):
     def __init__(self):
         """
         Manage cluster operation
         """
-        self._execute = SimpleCommand()
+        self._cluster_type = Conf.get(const.HA_GLOBAL_INDEX, "CLUSTER_MANAGER.cluster_type")
```

Comment:
Do we need any new fields in the confstore?

Reply:
Yes we need to detect env is it vm or hw
depending on this need to perform stonith operation.

Reply:
Please list the new fields here so that we can add them to the sheet from provisioner.

Reply:
in config
  "corosync-pacemaker": {
    "cluster_name": "cortx_cluster",
    "user": "hacluster",
    "secret": ""
}
Here corosync-pacemakerconsidered as type

Reply:
ok, these are already present.